workspaces/analytics/src/interfaces/Events.ts

```diff
@@ -0,0 +1,26 @@
+//@ts-ignore
+import keymirror from 'keymirror';
+export const Events = keymirror({
+  //on-boarding
+  ApiCreated: null,
+  ApiCheckCompleted: null,
+
+  //Errors
+  JavascriptErrorDetectedInFrontend: null,
+  RequirementForDiffsToHaveASuggestionFailed: null,
+
+  //running tasks
+  StartedTaskWithLocalCLI: null,
+  ExitedTaskWithLocalCLI: null,
+
+  //diffs
+  UserChangedCaptureOverviewTab: null,
+  NewBodyDiffRendered: null,
+  InferPolymorphismEnabled: null,
+  UserBeganAddingNewUrl: null,
+  UserFinishedAddingNewUrl: null,
+  DiffWasReset: null,
+  PreviewSuggestion: null,
```

Comment:
This one doesn't fit the grammar, and it seems like we're inconsistently applying the User prefix.